app/scripts/modules/kubernetes/src/v2/pipelines/stages/deleteManifest/deleteManifestStage.ts

```diff
@@ -1,24 +1,27 @@
 import { module } from 'angular';
 
-import { Registry, SETTINGS, IStage } from '@spinnaker/core';
+import { Registry, SETTINGS, IStage, ExecutionDetailsTasks } from '@spinnaker/core';
 import { KubernetesV2DeleteManifestConfigCtrl } from './deleteManifestConfig.controller';
-import { KUBERNETES_DELETE_MANIFEST_OPTIONS_FORM } from '../../../manifest/delete/deleteOptionsForm.component';
-import { KUBERNETES_MANIFEST_SELECTOR } from '../../../manifest/selector/selector.component';
+import { KUBERNETES_DELETE_MANIFEST_OPTIONS_FORM } from 'kubernetes/v2/manifest/delete/deleteOptionsForm.component';
+import { KUBERNETES_MANIFEST_SELECTOR } from 'kubernetes/v2/manifest/selector/selector.component';
+import { manifestExecutionDetails } from 'kubernetes/v2/pipelines/stages/traffic/ManifestTrafficExecutionDetails';
 
 export const KUBERNETES_DELETE_MANIFEST_STAGE = 'spinnaker.kubernetes.v2.pipeline.stage.deleteManifestStage';
 
+const STAGE_KEY = 'deleteManifest';
 module(KUBERNETES_DELETE_MANIFEST_STAGE, [KUBERNETES_DELETE_MANIFEST_OPTIONS_FORM, KUBERNETES_MANIFEST_SELECTOR])
   .config(() => {
     // Todo: replace feature flag with proper versioned provider mechanism once available.
     if (SETTINGS.feature.versionedProviders) {
       Registry.pipeline.registerStage({
         label: 'Delete (Manifest)',
         description: 'Destroy a Kubernetes object created from a manifest.',
-        key: 'deleteManifest',
+        key: STAGE_KEY,
         cloudProvider: 'kubernetes',
         templateUrl: require('./deleteManifestConfig.html'),
         controller: 'KubernetesV2DeleteManifestConfigCtrl',
         controllerAs: 'ctrl',
+        executionDetailsSections: [manifestExecutionDetails(STAGE_KEY), ExecutionDetailsTasks],
```

Comment:
Delete can delete multiple manifests when label selectors are used -- how is that rendered?

Reply:
updated to render the label selectors and the deleted manifest names

Reply:
SGTM